angular/playground/projects/ng-entity-service/src/lib/types.ts

```diff
@@ -28,3 +30,28 @@ export type HttpConfig<Entity = any> = {
   url?: string;
   mapResponseFn?: (res: any) => Entity | Entity[];
 };
+
+interface StoreWrite {
+  /**
+   * Disables writing to the store
+   *
+   * You then have to manually write to the store.
+   * This is useful when pairing the NgEntityService with the PaginatorPlugin
+   */
+  skip?: boolean;
+}
+
+export type HttpGetConfig<Entity = any> = HttpConfig<Entity> & {
+  append?: boolean; // TODO fix type these are mutually exclusive
```

Comment:
TODO?

Reply:
I didn't refactor this, so it's the same as before. I just left a todo as a reminder that the type is not 100% correct, as append and upsert are mutually exclusive